modules/local/alignpseudogenomes.nf

```diff
@@ -0,0 +1,49 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process ALIGNPSEUDOGENOMES {
+    label 'process_low'
```

Comment:
Here you already have the `labels` for the process resource requirements, that's perfect. So these are the ones that are specified in the `base.config`.

If a process has no label then the requested resources for the process will be the default (1 CPU, 1.GB).